packages/auth/src/signature/middleware.ts

```diff
@@ -21,12 +21,13 @@ function contextToRequestLike(ctx: AppContext): RequestLike {
 
 async function verifySigFromClient(
   client: string,
+  clientKeyId: string,
   ctx: AppContext
 ): Promise<boolean> {
   const clientService = await ctx.container.use('clientService')
   const clientKey = await clientService.getKey({
     client,
-    keyId: ctx.clientKeyId
+    keyId: clientKeyId
   })
 
   if (!clientKey) {
```

Comment:
Maybe we can consolidate our `ctx.clientKeyId = clientKeyId` assignments to below this check

Reply:
What do you mean by "below this check"?